triggers/command.py

```diff
@@ -0,0 +1,9 @@
+class CommandTrigger():
```

Comment:
I love the idea of a command-based trigger, that's awesome. Is there any reason we can't put the command decorator on the trigger itself though? It feels weird to have to do it to every block that has this trigger. If not, we may want to look into a different way to expose the command then rather than the decorator.

Reply:
this was @hansmosh 's idea and block, i just made the PR! I'll look into it, thanks for the feedback!